core/iwasm/compilation/aot_llvm.c

```diff
@@ -748,8 +748,9 @@ aot_add_llvm_func(AOTCompContext *comp_ctx, LLVMModuleRef module,
              * should be ok. We prefer short-call because it's smaller
              * and more importantly doesn't involve relocations.
              */
-            LLVMAttributeRef attr_short_call = LLVMCreateStringAttribute(
-                comp_ctx->context, "short-call", strlen("short-call"), "", 0);
+            LLVMAttributeRef attr_short_call =
+                LLVMCreateStringAttribute(comp_ctx->context, "short-call",
+                                          (uint32)strlen("short-call"), "", 0);
```

Comment:
i guess this should be unsigned, not uint32.

Reply:
yes, the related parameter of LLVMCreateStringAttribute is `unsigned KLength` indeed.